ultralytics/models/yolo/pose/val.py

```diff
@@ -64,39 +64,51 @@ def get_desc(self):
         )
 
     def postprocess(self, preds, img_shape):
-        """Apply non-maximum suppression and return detections with high confidence scores."""
         if self.separate_outputs:  # Quant friendly export with separated outputs
-            pred_order, nkpt = separate_outputs_decode(preds, self.args.task)
+            #mcv = float('-inf')
```

Comment:
???? Why have you changed pose val

Reply:
This was part of an experiment, and I forgot to revert these changes, I apologize. I will change them back.